docassemble/assemblylinewizard/generator_constants.py

```diff
@@ -14,7 +14,12 @@ class Object(object):
   #'attorney_of_record_address_on_one_line',
 ]
 
-# Vars representing people
+# Singular person object prefixes that we deliberately leave undefined
```

Comment:
The name is a little confusing since the only variable we're using isn't a person, but a court. Maybe fixable just by dropping a comment here.

Reply:
Changed this. Not sure the name is much better. Maybe even a little worse, but in writing the comment, I realized there was more nuance there than was expressed.